tedge/src/cli/mqtt/subscribe.rs

```diff
@@ -0,0 +1,86 @@
+use crate::cli::mqtt::MqttError;
+use crate::command::{Command, ExecutionContext};
+use crate::utils::signals;
+use futures::future::FutureExt;
+use mqtt_client::{Client, Message, MessageStream, QoS, TopicFilter};
+use tokio::{io::AsyncWriteExt, select};
+
+pub struct MqttSubscribeCommand {
+    pub topic: String,
+    pub qos: QoS,
+    pub no_topic: bool,
```

Comment:
what is the meaning of `no_topic`? Try to find a better way. I have to look at the source code to understand the meaning of it.

Reply:
`print_topic`, `include_topic_in_output`? Try to use positive logic.

Reply:
`hide_topic` maybe. I now saw that it refers to the CLI option `--no-topic`. But in the code it's really hard to understand it's meaning

Reply:
Changed to `hide_topic`